e2e/etcd_test.go

```diff
@@ -433,9 +433,9 @@ func (ep *etcdProcess) Stop() error {
 }
 
 func (ep *etcdProcess) waitReady() error {
-	readyStrs := []string{"enabled capabilities for version", "published"}
+	readyStrs := []string{"starting server...", "published"}
```

Comment:
OK. Will drop this commit in this pullreq and submit a separated one for this issue. Thanks a lot for your comments.